Ports/.port_include.sh

```diff
@@ -312,13 +312,9 @@ do_download_file() {
 
 fetch_simple() {
     url="${1}"
-    filename="${2}"
-    checksum="${3}"
+    checksum="${2}"
 
-    if [ "${filename}" != "$(basename "${url}")" ]; then
-        echo "Filename '${filename}' does not match basename of '${url}'"
-        exit 1
-    fi
+    filename="$(basename "${url}")"
```

Comment:
This will produce exceptionally _shit_ names with URLs like https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub (`?p=config.git;a=blob_plain;f=config.sub`)
Getting the real file name will require sending a HEAD request and parsing `Content-Disposition` (see pro.cpp)

Reply:
Though I suppose none of the current ports have bad URLs, so this could be solved later if needed (after giving someone a headache)

Reply:
I primarily wanted to get it out of the way before implementing the new shiny thing. If it is a problem in the future, we can add some syntax for specifying a filename after the main thing that we are working towards is done.

We don't have any files currently whose names are particularly annoying, the most annoying thing is a few ports where we blindly fetch the main branch and therefore end up with `main.tar.gz` as a filename (which still isn't _that_ bad, since all files are in per-port folders). Ideally, most of these are gone anyways once we have proper support for Git.

Reply:
sure, sounds good.